.github/workflows/ci.yaml

```diff
@@ -92,4 +92,8 @@ jobs:
     - run: make WHAT=test/e2e/e2e.test
       working-directory: ./kubernetes
     - run: WORKDIR=/tmp/e2e hack/run-sharded-kcp.sh
-    - run: WORKDIR=/tmp/cluster hack/run-cluster-kcp.sh
\ No newline at end of file
+    - run: WORKDIR=/tmp/cluster hack/run-cluster-kcp.sh
+    - uses: actions/upload-artifact@v2
```

Comment:
If you add:

```
if: ${{ always() }}
```

...this will always upload the artifact even if previous steps failed.

Reply:
Good catch!